src/__tests__/shared/components/navbar/Navbar.test.tsx

```diff
@@ -101,107 +103,76 @@ describe('Navbar', () => {
+        setup(allPermissions)
+        history.location.pathname = '/enterprise-1701'
+        expect(history.location.pathname).not.toEqual('/')
+        userEvent.click(screen.getByText(/hospitalrun/i))
+        expect(history.location.pathname).toEqual('/')
+      })
     })
-  })
 
-  describe('add new', () => {
-    it('should show a shortcut if user has a permission', () => {
-      const wrapper = setup(allPermissions)
-      const hospitalRunNavbar = wrapper.find(HospitalRunNavbar)
-      const addNew = hospitalRunNavbar.find('.nav-add-new')
-      const { children } = addNew.first().props() as any
+    describe('add new', () => {
+      it('should show a shortcut if user has a permission', () => {
+        setup(allPermissions)
+        const navButton = screen.getByRole('button')
+        userEvent.click(navButton)
 
-      const [icon, label] = children[0].props.children
+        expect(
+          screen.getByRole('button', {
+            name: /patients\.newpatient/i,
+          }),
+        ).toBeInTheDocument()
 
-      expect(icon.props.icon).toEqual('patient-add')
-      expect(label).toEqual('patients.newPatient')
-    })
-
-    it('should not show a shortcut if user does not have a permission', () => {
-      // exclude labs and incidents permissions
-      const wrapper = setup(cloneDeep(allPermissions).slice(0, 6))
-      const hospitalRunNavbar = wrapper.find(HospitalRunNavbar)
-      const addNew = hospitalRunNavbar.find('.nav-add-new')
-      const { children } = addNew.first().props() as any
-
-      children.forEach((option: any) => {
-        expect(option.props.children).not.toEqual('labs.requests.new')
-        expect(option.props.children).not.toEqual('incidents.requests.new')
-        expect(option.props.children).not.toEqual('imagings.requests.new')
+        // 0 & 1 index are dashboard fixed elements, 2 index is first menu label for user
+        expect(screen.queryAllByRole('button')[2]).toHaveTextContent(/patients\.newpatient/i)
       })
     })
-  })
 
-  describe('account', () => {
-    it("should render a link with the user's identification", () => {
-      const expectedUserName = `user.login.currentlySignedInAs ${userName.givenName} ${userName.familyName}`
+    describe('account', () => {
+      it("should render a link with the user's identification", () => {
+        const expectedUserName = `user.login.currentlySignedInAs ${userName.givenName} ${userName.familyName}`
 
-      const wrapper = setup(allPermissions, userName)
-      const hospitalRunNavbar = wrapper.find(HospitalRunNavbar)
-      const accountLinkList = hospitalRunNavbar.find('.nav-account')
-      const { children } = accountLinkList.first().props() as any
+        setup(allPermissions, userName)
+        const navButton = screen.getByRole('button')
 
-      expect(children[0].props.children).toEqual([undefined, expectedUserName])
-    })
+        screen.debug(undefined, Infinity)
```

Comment:
Is there a need for debug to be committed?

Reply:
PR is in progress still